components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -117,8 +119,8 @@ export default class ConfirmIntegration extends React.Component {
                         }}
                     />
                     <CopyText
-                        idMessage='integrations.copy_client_secret'
-                        defaultMessage='Copy Client Secret'
+                        idMessage='integrations.copy_webhook_url'
+                        defaultMessage='Copy webhook URL'
```

Comment:
FYI, I'm just asking from UX whether this tooltip text should be in title case or sentence case. Seems we're currently using both in the UI, but title case more often.

https://community.mattermost.com/core/pl/xpd6naptuidfdfb1n5yw8i1dah

Reply:
Sure, would you like me to change this now, for 5.11? I just caught that it was wrongly cut&pasted from a different link.

Reply:
Can we move the hover to the RHS of the button so it doesn't block the text above?

![image](https://user-images.githubusercontent.com/7575921/56814442-d7220680-6804-11e9-9a74-c09095efc067.png)

Reply:
The proposal from UX is to use `Copy webhook URL` as included in the PR. So no changes required.

@jfrerich Possibly, but that will be a larger change as the tooltip is used elsewhere in the UI, such as below. But I encourage you to post it in [UX Design channel](https://community.mattermost.com/core/channels/ux-design) for consideration

![image](https://user-images.githubusercontent.com/13119842/56815700-012cf600-6810-11e9-9b29-30238a59b72b.png)